keepalived/vrrp/vrrp_parser.c

```diff
@@ -461,6 +461,7 @@ vrrp_int_handler(vector_t *strvec)
 		return;
 	}
 
+	strcpy(vrrp->ifname, name);
```

Comment:
strncpy() would be safer

Reply:
There is a check just above: if (strlen(name) >= IFNAMSIZ), I did like in vrrp_vmac_handler().

Please let me know if you still want me to update the pull request with strncpy(dst, src, sizeof(dst)), or with snprintf(dst, sizeof(dst), "%s", src)

Thanks for the fast review!